database/cassandra/cassandra.go

```diff
@@ -35,11 +36,33 @@ type Cassandra struct {
 	session  *gocql.Session
 	isLocked bool
 
-	// Open and WithInstance need to guarantee that config is never nil
+	// Open and WithSession need to guarantee that config is never nil
 	config *Config
 }
 
-func (p *Cassandra) Open(url string) (database.Driver, error) {
+func WithSession(session *gocql.Session, config *Config) (database.Driver, error) {
```

Comment:
for consistency with all the other drivers, shouldn't the name remain as `WithInstance`?

Reply:
Agreed, use the name `WithInstance` for consistency